packages/SwingSet/test/test-vpid-liveslots.js

```diff
@@ -473,13 +482,13 @@ async function doVatResolveCase23(t, which, mode, stalls) {
 
   // The VPIDs in the remaining messages will depend upon whether we retired
   // p1 during that resolution.
-  const RETIRE_VPIDS = false;
+  const RETIRE_VPIDS = true;
```

Comment:
move the mode check up here: we're retiring VPIDs if we're in the right mode, and then we can switch solely on `RETIRE_VPIDS` later